recipes/adam/meta.yaml

```diff
@@ -15,16 +15,16 @@ build:
 
 source:
   fn: adam-distribution-spark2_2.11-{{ version }}-bin.tar.gz
-  url: https://search.maven.org/remotecontent?filepath=org/bdgenomics/adam/adam-distribution-spark2_2.11/0.22.0/adam-distribution-spark2_2.11-0.22.0-bin.tar.gz
-  sha256: 31624954eb473f2ec24354af7e6303a47887cf2ca4076bb71f54618aef59e15b
+  url: https://search.maven.org/remotecontent?filepath=org/bdgenomics/adam/adam-distribution-spark2_2.11/0.23.0/adam-distribution-spark2_2.11-0.23.0-bin.tar.gz
```

Comment:
Since we have {{ version }}, using it in the URL would be good. Makes the next update diff smaller.